docs/releases/4.1.txt

```diff
@@ -289,6 +289,9 @@ Internationalization
 * The :func:`~django.conf.urls.i18n.i18n_patterns` function now supports
   languages with both scripts and regions.
 
+* The :djadmin:`makemessages` commands, :file:`.po` files will not change if there is
+  no change in translations. Formerly POT-Creation-Date was always updated.
```

Comment:
This should be moved to the [_"Backwards incompatible changes in 4.1" -> "Miscellaneous"_](https://docs.djangoproject.com/en/4.1/releases/4.1/#miscellaneous).